src/docs/frontend/using-hooks.mdx

```diff
@@ -20,61 +20,38 @@ We don't prefer libraries that use or don't use hooks. Instead favor libraries
 with cleaner, simpler APIs, and smaller bundle sizes over ones with larger more
 complex APIs or larger bundle sizes.
 
-
 ### Using react's built-in hooks
 
-The `useState`, `useMemo`, `useCallback`, `useContext`, and `useRef` hooks are welcome
+The `useState`, `useMemo`, `useCallback`, `useContext`, `useReducer` and `useRef` hooks are welcome
 in any functional component. They are generally a good choice in presentational
 components that need a small amount of state, or access to react primitives like
 references and context. For example, a component with a slide-out or expandable
 state.
 
 The `useEffect` hook is more complex and you need to be careful to track your
 dependencies and ensure that subscriptions are canceled through a cleanup
-callback. Complex chained applications of `useEffect` should be avoided and
-'controller' components should remain class-based at this time.
-
-Likewise, the `useReducer` hook overlaps with state-management which currently
-has an undecided future. We would like to avoid *yet another* state management
-pattern, so at this time avoid `useReducer`.
```

Comment:
Unsubstantiated claim about an `undecided future`.

`useReducer` looks solid: https://react.dev/reference/react/useReducer
things like `useOptimistic` looks like they're still experimental: https://react.dev/reference/react/useOptimistic
I don't see deprecation markers, and don't expect it for this one.